src/ipyradiant/visualization/interactive_exploration.py

```diff
@@ -89,21 +93,35 @@ class GetOutgoingPredicateObjects(SPARQLQueryFramer):
 
     """
 
-
-# Throughout this class, we assign the layout to self.cyto_graph_layout multiple times.
-# This is so that the graph refreshes the layout every time nodes are added or removed,
-# which provides an optimal viewing experience.
+    @classmethod
```

Comment:
This is not in master. I think it got added back during merge conflict resolution
https://github.com/jupyrdf/ipyradiant/blob/master/src/ipyradiant/visualization/interactive_exploration.py

Reply:
c020b69